arbitrum-docs/devs-how-tos/how-to-estimate-gas.mdx

```diff
@@ -1,20 +1,22 @@
 ---
-id: how-to-estimate-gas
 title: 'How to estimate gas in Arbitrum'
 description: Learn how to estimate gas before submitting transactions.
 author: TucksonDev
-content-type: how-to
+content_type: how-to
 ---
 
-import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md';
+:::info Looking for Stylus guidance?
```

Comment:
In cases where a core dev doc overlaps with a Stylus doc, I've pointed from one to the other, to mitigate discoverability and findability issues.